### PR TITLE
Clarify that separate error and audit is required for integrations

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: servicecontrol/contracts.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ To subscribe to the `MessageFailed` event:
 - Customize the endpoint configuration to use `JsonSerializer` as the message published by ServiceControl uses Json serialization
 - Also customize the endpoint configuration such that the following conventions are used, as the `MessageFailed` event that is published by ServiceControl does not derive from `IEvent`. 
 
+NOTE: It's important that your integration endpoint do not use the same `error` and `audit` queue as your "business" endpoints since this might risk failures in the integration endpoint to cause an infinite feedback loop. Audited messages can cause confusion when searching for messages using ServiceInsight. Details on how to adjust error queue settings can be found [here](/nservicebus/errors/) and audit queue settings [here](nservicebus/operations/auditing.md).
```
Comment (SimonCropp) [inline servicecontrol/contracts.md]: should `nservicebus/operations/auditing.md` be fully qualified?

Comment (SimonCropp) [inline servicecontrol/contracts.md]: also is `here` links the best u can do?

Comment (andreasohlund) [inline servicecontrol/contracts.md]: Link fixed

Comment (andreasohlund) [inline servicecontrol/contracts.md]: Do we have a standard? "See xyf for more details" 
Not 100% sure what you're getting at?

File: servicecontrol/contracts.md [hunk 2]
```diff
@@ -41,6 +41,8 @@ To subscribe to the `MessageFailed` event:
 - Customize the endpoint configuration to use `JsonSerializer` as the message published by ServiceControl uses Json serialization
 - Also customize the endpoint configuration such that the following conventions are used, as the `MessageFailed` event that is published by ServiceControl does not derive from `IEvent`. 
 
+NOTE: It's important that your integration endpoint do not use the same `error` and `audit` queue as your "business" endpoints since this might risk failures in the integration endpoint to cause an infinite feedback loop. Audited messages can cause confusion when searching for messages using ServiceInsight. Details on how to adjust error queue settings can be found [here](/nservicebus/errors/) and audit queue settings [here](/nservicebus/operations/auditing.md).
```
Comment (SimonCropp) [inline servicecontrol/contracts.md]: how about

    See also [adjust error queue settings](/nservicebus/errors/) and [audit queue settings](/nservicebus/operations/auditing.md).

Comment (andreasohlund) [inline servicecontrol/contracts.md]: Like it, fixed

File: servicecontrol/contracts.md [hunk 3]
```diff
@@ -41,6 +41,8 @@ To subscribe to the `MessageFailed` event:
 - Customize the endpoint configuration to use `JsonSerializer` as the message published by ServiceControl uses Json serialization
 - Also customize the endpoint configuration such that the following conventions are used, as the `MessageFailed` event that is published by ServiceControl does not derive from `IEvent`. 
 
+NOTE: It's important that your integration endpoint do not use the same `error` and `audit` queue as your "business" endpoints since this might risk failures in the integration endpoint to cause an infinite feedback loop. Audited messages can cause confusion when searching for messages using ServiceInsight. See also [adjust error queue settings](/nservicebus/errors/) and [audit queue settings](/nservicebus/operations/auditing.md).
```
Comment (SimonCropp) [inline servicecontrol/contracts.md]: > It's important that your integration endpoint do not use the same

should that be "endpoints" or "does not"

Comment (SimonCropp) [inline servicecontrol/contracts.md]: > Audited messages can cause confusion when searching for messages using ServiceInsight

this is lacking context

**Comment by andreasohlund** [comment]
@SimonCropp can you look now?

**Pull request merged**